internal/github/github.go

```diff
@@ -62,6 +62,8 @@ func (r *RealGitHub) CreatePullRequest(output io.Writer, workingDir string, pr P
 	if strings.Contains(execOutput, "GraphQL error: No commits between") {
 		// no PR was created because there are no differences between remotes
 		return false, nil
+	} else if strings.HasPrefix(execOutput, "Dry-run mode") {
```

Comment:
Would there be a reason to not just use the value of `flags.DryRun` here? That way we'd avoid the need to keep strings in sync.

Reply:
Done in https://github.com/Skyscanner/turbolift/pull/88/commits/d96c6b0c83237a2832b2c1940640d3dab7288192